grammar.js

```diff
@@ -815,16 +815,34 @@ module.exports = grammar({
         ),
       ),
 
+    // Any ascii non-blanck printable character (no unicode)
+    _syn_keyword_identifier: ($) =>
+        choice(
+          seq(/[!-~]/, repeat(token.immediate(/[!-~]/))),
```

Comment:
This looks really weird, why not a simple regex ?

Reply:
Maybe it comes from a misunderstanding of priorities for me, but if I use a simple regex, the result is not the same as
regex has a lower priority. Adding manually a higher priority with `prec(*)` doesn't give the same result either.

Reply:
If you find how to do it properly I would be glad to see it cause I have spent probably two hours trying to make it work

Reply:
I'll try to find a way, but that worries me a litte.